### DISCUSSION TITLE
What is the real difference between this project and plain old Makefile?

### DISCUSSION THREAD

**Discussion by woss** [sagiegurari/cargo-make#577] [Q&A] [answered]
I'm researching how to automate my project and I came across this project and it seems interesting. Although I am not sure what the added benefit is over the standard `Makefile`.

**Comment by sagiegurari** [comment, marked as answer]
make is very different.
there are many added values in cargo-make, such as built in watch, cross platform, ability to extend/load other makefiles, conditions on tasks, and a lot more.
There are additional big advantages for rust projects such as: integration with cargo/rustup to install dependencies, ability to choose nightly/beta/stable, predefined env vars to help you understand the rust env and so on...
